broadcaster/_backends/redis.py

```diff
@@ -34,3 +37,20 @@ async def publish(self, channel: str, message: typing.Any) -> None:
     async def next_published(self) -> Event:
         message = await self._subscriber.next_published()
         return Event(channel=message.channel, message=message.value)
+
+    def parse_query_db_password(self, query: str) -> None:
```

Comment:
https://stackoverflow.com/questions/10113090/best-way-to-parse-a-url-query-string

urlib.parse might be a better option rather than hand parsing the querystring as well.